spec/features/adding_spaces_spec.rb

```diff
@@ -5,8 +5,10 @@
     expect(current_path).to eq "/spaces/new"
 
     fill_in :space_name, with: "Cozy Cottage"
+    fill_in :description, with: "Nice walks near by"
     click_button("Submit")
 
     expect(page).to have_content("Cozy Cottage")
+    expect(page).to have_content("Nice walks near by")
   end
```

Comment:
K&M understood - good to remove comments now - discuss whether comments should be left in for future developers?